### PR TITLE
Remove dynamic in Package.swift

### PR DESCRIPTION
**Pull request by sacheu**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -37,8 +37,6 @@ let package = Package(
     products: [
         .library(name: "AzureCore", targets: ["AzureCore"]),
```
Comment (raosanat) [inline Package.swift:38]: this needs to be dynamic as well ?

**Review by tjprescott — CHANGES_REQUESTED** [review]
Awaiting approval from @raosanat.

File: Package.swift (hunk 1 above)
Comment (sacheu) [reply Package.swift:38]: Sanath is testing without making this dynamic.

**Review by tjprescott — APPROVED** [review]

File: Package.swift (hunk 1 above)
Comment (sacheu) [reply Package.swift:38]: It is removed.

**Pull request merged**